docs/tianocore_build.md

```diff
@@ -0,0 +1,89 @@
+# Coreboot with tianocore payload on apu2
+
+This documentation describes how to build coreboot image with tianocore payload
+for PC Engines apu2 platform.
+
+## Building coreboot image
+
+1. Download coreboot-sdk docker image
+```
+docker pull coreboot/coreboot-sdk:1.52
+```
+
+2. Run docker container with passing coreboot project directory
+```
+docker run --rm -it -v <coreboot-directory>:/home/coreboot coreboot/coreboot-sdk:1.52 /bin/bash
+```
+
+  Notice that `<coreboot-directory>` is your local directory with coreboot
+  project. `/home/coreboot` will be directory for that project in docker
+  container. Therefore, it's up to you what path to give.  
+
+3. In docker container go to coreboot main folder
+```
+coreboot@32ceb7327125:/$ cd home/coreboot
+coreboot@32ceb7327125:~$ ls
+3rdparty  CHANGELOG.md	COPYING  Documentation	MAINTAINERS  Makefile  
+Makefile.inc  README.md	build  configs	gnat.adc  payloads  src  toolchain.inc
+util
+```
+
+4. Run distclean
+```
+coreboot@32ceb7327125:~$ make distclean
+```
+
+5. Copy config file for target platform
+```
+coreboot@32ceb7327125:~$ cp configs/config.pcengines_apu2 .config
+```
+
+6. Run olddefconfig
+```
+coreboot@32ceb7327125:~$ make olddefconfig
+```
+
+7. Run menuconfig
+```
+coreboot@32ceb7327125:~$ make menuconfig
+```
+
+8. In menuconfig go to `Payload` menu and next:
+
+  a. In `Add a payload` choose *Tianocore coreboot payload package*
+
+  b. Select tianocore build type (debug or release)
```

Comment:
release only. User probably won't like the debug output

Reply:
Done